src/MegaPoker.sol

```diff
@@ -18,104 +18,91 @@
 
 pragma solidity ^0.6.11;
```

Comment:
Should we go to 0.6.12 here? We seem to be slowly bumping up to that across our repos.

Reply:
Yes, will change it.

Reply:
Done